prov/util/src/util_attr.c

```diff
@@ -1093,6 +1095,13 @@ static void fi_alter_domain_attr(struct fi_domain_attr *attr,
 						attr->mr_mode & hints_mr_mode);
 		}
 	}
+}
```

Comment:
Please rename to ofi_alter_mr_mode.  We only need to pass in the mr_mode bits, versus the domain attributes.
```
ofi_alter_mr_mode(int *mr_mode, hints, info_caps, api_version);
```

Reply:
Done